types/module/server/testutil.go

```diff
@@ -28,9 +28,10 @@ type FixtureFactory struct {
 	signers []sdk.AccAddress
 	cdc     *codec.ProtoCodec
 	baseApp *baseapp.BaseApp
+	invar   *sdk.InvariantRegistry
 }
 
-func NewFixtureFactory(t *testing.T, numSigners int) *FixtureFactory {
+func NewFixtureFactory(t *testing.T, numSigners int, invariant sdk.InvariantRegistry) *FixtureFactory {
```

Comment:
why adding `invariant` as param here? It's always set as `nil` in the tests so far. On that note, it would be good to add some tests for the group invariants.

Reply:
Thanks, we'll do necessary changes